po/da.po

```diff
@@ -42,7 +42,7 @@ msgid ""
 "Mount/unmount filesystems defined in the fstab file with the x-udisks-auth "
 "option"
 msgstr ""
-"Monter/afmonter filsystemer defineret i fstab-filen med tilvalget x-storaged-"
```

Comment:
It seems better to me to actually do this. First of all, the current *.po files are wrong: They translate `x-udisks-...` to `x-storaged-...`, probably a victim of the earlier rename in the other direction. Second, if we can fix them mechanically I don't see the need of imposing manual labor on translators. And thirdly, as the `msgid` did not actually change, the POT file will not change either, so this wouldn't even do anything.

However, one instance of translating "Intel Rapid Storaged Technology" crept in, due to the typo (the 'd' should not be there). I'll fix that properly.